src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
Because the error is occurring due to absence of `int32_t`,  maybe I could change the type to `i64`, since previous examples have used it, and obviously they have passed all the tests/builds. what do you think?